### ISSUE TITLE
Broken download links for non-standard systems (macos,linux etc)

### ISSUE THREAD

**Issue by gauravphoenix** [ClickHouse/ClickHouse#33480]
The links listed [here](https://clickhouse.com/docs/en/getting-started/install/#from-binaries-non-linux) are broken:

```bash
curl  https://builds.clickhouse.com/master/macos/clickhouse
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Resource>/master/macos/clickhouse</Resource><RequestId>844a0a06751ed008</RequestId></Error>%
```

**Comment by chetanmelkani** [comment]
are there any alternatives till this is fixed?

**Comment by Felixoid** [comment]
Yes, for any commit in `master` you can click on checks mark :heavy_check_mark: or :x:, and find a report called `ClickHouse Special Build Check (actions)`

For example, here you can find it for the recent commit https://s3.amazonaws.com/clickhouse-builds/0/e6ea7c361d2f117ded5785acbcf59764881e4bd9/clickhouse_special_build_check_(actions)/report.html

All special builds are there.

I'll fix the links soon

**Comment by Felixoid** [comment]
Three points are here:

- [ ] Push the special builds artifacts to static URLs, like `https://s3.amazonaws.com/clickhouse-builds/master/binary_darwin/clickhouse`, `https://s3.amazonaws.com/clickhouse-builds/master/binary_darwin_aarch64/clickhouse` etc
- [ ] Add the worker to https://github.com/ClickHouse/clickhouse-website-worker/ for builds.clickhouse.com
- [ ] Fix the links in documentation
